frontend/amundsen_application/static/js/components/ResourceListItem/FeatureListItem/index.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import { getSourceDisplayName, getSourceIconClass } from 'config/config-utils';
+import { logClick } from 'utils/analytics';
+
+import BadgeList from 'features/BadgeList';
+import { ResourceType, FeatureResource } from 'interfaces';
+
+import { LoggingParams } from '../types';
+
+export interface FeatureListItemProps {
+  feature: FeatureResource;
+  logging: LoggingParams;
+}
+
+class FeatureListItem extends React.Component<FeatureListItemProps, {}> {
```

Comment:
We are trying to move out of the class syntax for new work, could you try to get this as a functional component?